subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/JavaCompileIntegrationTest.groovy

```diff
@@ -544,15 +544,15 @@ class JavaCompileIntegrationTest extends AbstractPluginIntegrationTest {
         // This file has a file name which is not UTF-8.
         // See https://bugs.openjdk.java.net/browse/JDK-7062777?focusedCommentId=12254124&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-12254124.
         resources.findResource('broken-utf8.is-a-jar').copyTo(file('broken-utf8.jar'))
-        file('src/main/java/Hello.java') << 'public class Hello {}'
+        file('src/test/java/Hello.java') << 'public class Hello {}'
         executer.withStackTraceChecksDisabled()
 
         when:
-        run 'compileJava', '--debug'
+        run 'test', '--debug'
```

Comment:
I had to change this test, because `@CompileClasspath` now truly ignores resources - It doesn't even try to inflate them anymore.